projects/video-player-wc/src/test.ts

```diff
@@ -17,7 +17,9 @@ declare const require: {
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting()
+  platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
```

Comment:
Why this is false? it will increase the test case running time

Reply:
Angular migration itself added this , manually i have not changed.